doc/source/reference/random/bit_generators/index.rst

```diff
@@ -105,6 +105,35 @@ If you need to generate a good seed "offline", then ``SeedSequence().entropy``
 or using ``secrets.randbits(128)`` from the standard library are both
 convenient ways.
 
+If you need to run several stochastic simulations in parallel, best practice
+is to construct a random generator instance for each simulation. To make sure
+the random streams are independent, you can use the `spawn` method of
+`~SeedSequence`. For instance, here we construct a list of 12 independent
+generator instances:
+
+.. code-block:: python
+
+    from numpy.random import PCG64, SeedSequence
+    base_seq = SeedSequence(1234)
+    child_seqs = base_seq.spawn(12)    # a list of 12 SeedSequences
+    generators = [PCG64(seq) for seq in child_seqs]
+
+.. end_block
+
+
+An alternative way is to use the fact that a `~SeedSequence` can be initialized
```

Comment:
Should you mention that these are distinct?

Reply:
Distinct in the sense that even though they sort of look similar, they will not produce the same sequences.